_includes/templates/header-external-site.html

```diff
@@ -0,0 +1,20 @@
+<header class="header
```

Comment:
Yes, that's a very good point! Fixed in https://github.com/cfpb/cfgov-refresh/commit/00f0cd8ee973fed947838f7744a78faeabbd1f4e or it could be the elegant `header-sans-nav.html` ;-)